src/include/sof/drivers/cache_attr.h

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Iuliana Prodan <iuliana.prodan@nxp.com>
+ */
+
+#ifdef CONFIG_COMPILER_WORKAROUND_CACHE_ATTR
+
+#ifndef __SOF_DRIVERS_CACHE_ATTR_H__
```

Comment:
The include guard should come before the CONFIG_COMPILER.....

Reply:
Done